rasa/api.py

```diff
@@ -20,7 +20,7 @@ def run(
     connector: "Text" = None,
     credentials: "Text" = None,
     **kwargs: "Dict[Text, Any]",
-) -> "NoReturn":
```

Comment:
Huh - I never knew about `NoReturn`. Seems to indicate a special case where a function only ever exits via Exception i.e. it literally never returns. Maybe we just want this without quotes? Or did the mypy check say it should be `None`?

Reply:
the thing is that the function implicitly returns. There is no `return` statement, but the function isn't raising an exception consistently. So mypy rightfully complains :)